blocknative/utils.py

```diff
@@ -34,31 +34,25 @@ def status_error_to_exception(message: dict) -> None:
         SDKError: If there is a non-specific error that is indicated by the server.
     """
 
-    if message["status"] == "ok" or "reason" in message:
+    if message['status'] == "ok" or 'reason' not in message:
```

Comment:
Let's do double quotes for consistency
```suggestion
    if message["status"] == "ok" or "reason" not in message:
```

Reply:
We should standardize on using single quotes as that is more 'pythonic'.   I will revert here but we should follow up in a later pr.